benchmarks/spinlock_bench/spinlock_bench.c

```diff
@@ -0,0 +1,102 @@
+/****************************************************************************
+ * apps/benchmarks/spinlock_bench/spinlock_bench.c
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.  The
+ * ASF licenses this file to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the
+ * License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ ****************************************************************************/
+
+/****************************************************************************
+ * Included Files
+ ****************************************************************************/
+
+#include <stdio.h>
+#include <assert.h>
+#include <errno.h>
+#include <nuttx/spinlock.h>
+#include <time.h>
+
+/****************************************************************************
+ * Pre-processor Definitions
+ ****************************************************************************/
+
+#define TEST_NUM CONFIG_SPINLOCK_MULTITHREAD
+#define THREAD_NUM CONFIG_SPINLOCK_MULTITHREAD
+
+/****************************************************************************
+ * Private Data
+ ****************************************************************************/
+
+struct thread_parmeter_s
+{
+  int *g_result;
+  spinlock_t *lock;
+};
+
+static FAR void *thread_spinlock(FAR void *parameter)
```

Comment:
let's use FAR consistent across the code base.

Reply:
Im not sure every type is needed to add FAR or not.
If there is any mistake, please tell me.

Reply:
pointer in common code need add FAR, except the arch/chip/board specific code could remove FAR if they don't distinguish near and far.

Reply:
> pointer in common code need add FAR, except the arch/chip/board specific code could remove FAR if they don't distinguish near and far.

Got it. Thanks your answer.